src/components/Square.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Square(props) {
```

Comment:
One small note: you can use object destructuring if you like to avoid needing to call `props.` multiple times on the next line. `function Square({click, value})`